### PR TITLE
Weather code tree update

### PR DESCRIPTION
**Pull request by MoseleyS**
Addresses https://github.com/metoppv/mo-blue-team/issues/92

Following work on the issue above, I found that the weather code inputs needed updating. As the weather code decision trees have recently been changed to include hail and to work with precipitation accumulation data instead of rate, I chose to update the acceptance tests with the new decision trees and fresh data. The new acceptance test data are in branch `mobt_92_wxcode_tree_update`.

As the new data have no lightning symbols in them, I artificially inflated the lightning data to ensure there is a difference between the with-lightning and without-lightning tests.

Testing:
 - [x] Ran tests and they passed OK

### CODE REVIEW

File: improver_tests/acceptance/SHA256SUMS [hunk 1]
```diff
@@ -700,37 +700,42 @@ c798e3530eac9a45305711d5e91daa56d5005b826ce85bb48e2889e328ea965c  ./wxcode-modal
...
-4daea2d64e5b55507b7d0cda37642a08e265e89e5f72584845182e45ffa37eab  ./wxcode/native_units/probability_of_lwe_thickness_of_sleetfall_amount_above_threshold.nc
-8699b5267ecfdba1259d4b3d0a9a7ef33b75838d5ded75d041d48938a556a5fa  ./wxcode/native_units/probability_of_lwe_thickness_of_snowfall_amount_above_threshold.nc
-a5e061dd1d1b476f4e9bf2d959a7ee267ceaeb41bc0ddb4f95e0208b2c267a37  ./wxcode/native_units/probability_of_shower_condition_above_threshold.nc
-0ebf269ce0bd8bdb87322589163ec266c7e3b4fa62133e6db840910d942ad41e  ./wxcode/native_units/probability_of_thickness_of_rainfall_amount_above_threshold.nc
-9831c847f94f68b1a4917e9d68f123d54fc9f87447ff6d34fddf71bfa4737117  ./wxcode/native_units/probability_of_visibility_in_air_below_threshold.nc
-10a3502534d286f0abb732bc4a4bb0b9e494a073b2d572693e080d5e4633d66d  ./wxcode/wx_decision_tree_1h.json
-f2b140ab62972b5fe5e52ddef4550a9fde440da825070999c0a51d08f10d060f  ./wxcode/wx_decision_tree_3h.json
+5a3a808f0cc7952e6cc4225d09136276f685c5fdd6798a795c0430aa65ca71fa  ./wxcode/basic/kgo.nc
```
Comment (gavinevans) [inline improver_tests/acceptance/SHA256SUMS:703]: Just a couple of queries regarding these acceptance tests.

1. Based on [this comment](https://github.com/metoppv/mo-blue-team/issues/92#issue-1016463122), it seems like the intention is not to include "in_vicinity" within the cell method for the vicinity diagnostics. For the inputs to the weather symbols, it seems like they do not have "in_vicinity" in the cell method, which seems correct but ​`interpret-metadata`​ seems to think that they should. Will just behaviour of ​`interpret-metadata`​ be corrected, or has been corrected by one of your PRs e.g. https://github.com/metoppv/improver/pull/1632? 

2. ​`interpret-metadata`​ warns about the ​`mosg__model_run​` attribute, which is present on some but not all of the inputs. Should this attribute be present?

**Review by gavinevans — CHANGES_REQUESTED** [review]
Thanks for these amendments, @MoseleyS 👍 

I've added a query.

File: improver_tests/acceptance/SHA256SUMS (hunk 1 above)
Comment (gavinevans) [reply improver_tests/acceptance/SHA256SUMS:703]: OK, I've now realised that the `mosg__model_run` attribute is potentially expected after model blending as model provenance information following https://github.com/metoppv/improver/pull/1569. The presence of the `mosg__model_run` attribute on these inputs is, therefore, fine.

**Review by gavinevans — APPROVED** [review]

File: improver_tests/acceptance/SHA256SUMS (hunk 1 above)
Comment (gavinevans) [reply improver_tests/acceptance/SHA256SUMS:703]: Given that `mosg__model_run` is an accepted attribute, potentially `interpret_metadata` should be updated to reflect this.

**Comment by bayliffe** [comment]
Plotted it all. Ran the tests. Happy.

**Review by bayliffe — APPROVED** [review]

**Comment by MoseleyS** [comment]
Merged acceptance test data from `mobt_92_wxcode_tree_update`

**Pull request merged**